### ISSUE TITLE
How can I adjust the parameters to get better assembly results?

### ISSUE THREAD

**Issue by XXH123a** [chhylp123/hifiasm#416]
![plot](https://user-images.githubusercontent.com/96107726/224024372-39c68b53-3a9b-4d11-90b2-6bdf53661594.png)
![quast_result](https://user-images.githubusercontent.com/96107726/224025624-e0eaf4ef-a11d-4769-a47d-996f108b37c5.png)

Dear Dr. Cheng, Hello.
      Excuse me, I have assembled two haplotypes and *.p_ctg.fa with Hi-C+Hifi on a highly heterozygous plant genome and the three results are about 460MB, but I got 380M (similar to the survey result) with Hicanu, I don't understand how to adjust the "-D,-N,--hom-cov ", very much look forward to your guidance (two graphs are survey and quast results)

**Comment by chhylp123** [comment]
Are both haplotypes produced by hifiasm about 460Mb?

**Comment by chhylp123** [comment]
I guess maybe the size of hifiasm assembly is right? Purge_dups often makes the assembly smaller, which migh be wrong. Could you please check the BUSCO score first?

**Comment by XXH123a** [comment]
![ab47ab8d9a6a3a9dc959baa1e7ef8ed](https://user-images.githubusercontent.com/96107726/224523624-9597ec27-5473-4043-930e-b076dc213711.png)
Dear Dr. Cheng,
     Hifiasm is so excellent! Your view is very correct! As shown in the figure, the busco value is very good, so I can scaffold hap1 and hap2 separately (for example, 3d dna) to get the complete two sets of genomes. Do you think I am right to do this?

**Comment by chhylp123** [comment]
Yes. Another solution is to scaffold both haplotypes at the same time.
